src/lib/Actions.js

```diff
@@ -66,6 +66,17 @@ export async function migrateMakerDao() {
     }
 }
 
+export async function exportBackToMakerDao(onHash) {
+    try{
+        const exportCdpBackToMakerDao = B.exportFresh(web3, networkId, userInfo.proxyInfo.userProxy, userInfo.bCdpInfo.cdp, onHash)
```

Comment:
`onHash` is not part of the input, i guess it works because js is yoloing
but please remove.